nucleus/deploy/client.py

```diff
@@ -103,16 +103,20 @@ def register_endpoint_auth_decorator(self, endpoint_auth_decorator_fn):
     def create_model_bundle(
         self,
         model_bundle_name: str,
-        load_predict_fn: Callable[[DeployModel_T], Callable[[Any], Any]],
         env_params: Dict[str, str],
+        *,
+        load_predict_fn: Optional[
+            Callable[[DeployModel_T], Callable[[Any], Any]]
+        ] = None,
+        predict_fn_or_cls: Optional[Callable[[Any], Any]] = None,
         requirements: Optional[List[str]] = None,
         model: Optional[DeployModel_T] = None,
         load_model_fn: Optional[Callable[[], DeployModel_T]] = None,
         bundle_url: Optional[str] = None,
     ) -> ModelBundle:
         """
         Grabs a s3 signed url and uploads a model bundle to Scale Deploy.
-        A model bundle consists of a "load_predict_fn" and exactly one of "model" or "load_model_fn", such that
+        A model bundle consists of a "predict_fn_or_cls" or "load_predict_fn" and exactly one of "model" or "load_model_fn", such that
```

Comment:
I think the allowed combinations are  `{predict_fn_or_cls}, {load_predict_fn + model}, {load_predict_fn + load_model_fn}`, I'm not sure if it's completely clear from reading this. Maybe something like "A model bundle consists of {predict_fn_or_cls}, {load_predict_fn + model}, or {load_predict_fn + load_model_fn}.".